CHANGELOG.md

```diff
@@ -30,6 +30,9 @@
 - Proper `User-Agent` header is now set on outgoing HTTP requests.
   [#387](https://github.com/Kong/gateway-operator/pull/387)
 
+- Introduce `KongPluginInstallation` CRD
+  [400](https://github.com/Kong/gateway-operator/pull/400)
```

Comment:
This should maybe wait pending the rest of the feature, and then be a single changelog entry that mentions multiple PRs. Could also just leave this in and then update it as the other bits are added.

Reply:
My typical flow is to add it as fast as possible and updating then